calcom/cal.com
# Fix time format on booking page

## What does this PR do?



This PR fixes the 12/24hr time format on the booking page.

Fixes # (issue)


https://www.loom.com/share/63735a2317cb454196ae1bb8be5c9d00

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- On a private browser visit a booking page
	- Depending on your locale it should be in your locale time format
- In `timeFormat.ts` line 32 change a locale that has a time format opposite of yourself
- Delete  `timeOption.is24hClock` from localStorage
- Refresh the booking page
	- It should be in the opposite locale now 	

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
